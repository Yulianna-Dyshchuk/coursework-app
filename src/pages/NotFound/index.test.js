import React from "react";
import { shallow } from "enzyme";
import { NotFound } from "./index";

describe("NotFound component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<NotFound />);

    expect(component).toMatchSnapshot();
  });
});
