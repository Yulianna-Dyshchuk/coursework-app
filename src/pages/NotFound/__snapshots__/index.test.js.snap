// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render correctly without props 1`] = `
<main>
  <div>
    <WithStyles(ForwardRef(Typography))
      variant="h2"
    >
      404
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Typography))
      variant="body2"
    >
      Ooops... URL is not found! 
      <Link
        to="/"
      >
        Do you want to go to the home page?
      </Link>
    </WithStyles(ForwardRef(Typography))>
  </div>
</main>
`;
